cmd/erasure-utils.go

```diff
@@ -40,11 +41,19 @@ func newHashWriters(diskCount int, algo string) []hash.Hash {
 func newHash(algo string) hash.Hash {
 	switch algo {
 	case "blake2b":
-		return blake2b.New512()
+		h, err := blake2b.New512(nil)
+		if err != nil {
+			panic(fmt.Sprintf("impossible situation: %s", err))
```

Comment:
> Maybe something like: "used blake2b as MAC with an invalid key" or is ignoring the error the better behavior?

We can do this as well use blake2b.New512() from crypto if it returns an error , fall back to blake2b-simd project.